modules/dex-core/src/main/scala/org/ergoplatform/dex/domain/amm/CFMMVersionedOrder.scala

```diff
@@ -0,0 +1,70 @@
+package org.ergoplatform.dex.domain.amm
+
+import derevo.circe.{decoder, encoder}
+import derevo.derive
+import io.circe.{Decoder, Encoder}
+import org.ergoplatform.ergo.domain.Output
+import tofu.logging.derivation.loggable
+import io.circe.syntax._
+import cats.syntax.either._
+
+sealed trait CFMMVersionedOrder[+version <: CFMMOrderVersion, +order <: CFMMOrderType] {
```

Comment:
Lets remove "Versioned" from the name. It adds nothing to the semantics but makes the class name longer.